Templates/HttpTrigger-JavaScript/index.js

```diff
@@ -2,16 +2,16 @@
     context.log('JavaScript HTTP trigger function processed a request.');
 
     if (req.query.name || (req.body && req.body.name)) {
-        res = {
+        context.res = {
             // status: 200, /* Defaults to 200 */
             body: "Hello " + (req.query.name || req.body.name)
         };
+        context.done();
     }
     else {
-        res = {
-            status: 400,
-            body: "Please pass a name on the query string or in the request body"
-        };
+        // express-like response api
```

Comment:
Let's not mix these two different styles in the same template. I get what you're trying to achieve, but I'd like to keep the templates consistent.